OpenRA.Mods.Common/Traits/World/ActorSpawnManager.cs

```diff
@@ -0,0 +1,119 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Controls the spawning of actors. Attach this to the world actor.")]
+	public class ActorSpawnManagerInfo : ConditionalTraitInfo, Requires<MapCreepsInfo>
+	{
+		[Desc("Minimum number of actors")]
+		public readonly int Minimum = 0;
+
+		[Desc("Maximum number of actors")]
+		public readonly int Maximum = 4;
+
+		[Desc("Time (in ticks) between actor spawn.")]
+		public readonly int SpawnInterval = 6000;
+
+		[FieldLoader.Require]
+		[Desc("Name of the actor that will be spawned.")]
+		public readonly string[] ActorSignature = { };
+
+		public readonly string ActorSignNotification = "";
+		public readonly string ActorOwnerPlayer = "Creeps";
```

Comment:
I'd argue that this would be just as clear as "ActorOwner:"

Reply:
That was directly copied from WormManager, as with all the comments